meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn

```diff
@@ -12,19 +12,14 @@ if [ ! -f $CONFIG_PATH ]; then
 	exit 1
 fi
 
-while :; do
-	_uuid=$(jq -r '.uuid' $CONFIG_PATH)
-	# If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
-	_device_api_key=$(jq -r '.apiKey // .deviceApiKey' $CONFIG_PATH)
+# If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
+_device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Yep, that's the plan, however this is for the case where an old device has had a host update and hasn't yet done a key exchange so it only has a "provisioning key" (which is really a provisioning+device key) eg if supervisor hasn't been updated to one that supports key exchange or hasn't had a chance to do it yet